docs/guides/migrations.md

```diff
@@ -19,6 +19,15 @@ The OpenResty/NGINX image was updated to `openresty/openresty:1.27.1.2-2-jammy`
 This update resolves the issue with the `luarocks` package manager (see [#1825](https://github.com/intershop/intershop-pwa/pull/1825)).
 With the working `luarocks` package manager, the installation of the `lua-resty-redis-connector` was re-enabeld again.
 
+The order history listing for the PWA was improved.
+Previously, there was a "load more" button to extend the order list if there were more orders to display.
+Now, the order list provides a complete paging bar if there are more than 25 orders (by default) to display.
+This functionality requires ICM 13.1.0 or newer, which includes the extended Order REST API that returns the necessary paging information.
```

Comment:
That would be the Order REST API 1.8.0. Should we add this info / link the reference? (https://support.intershop.com/kb/index.php/Display/3T1454)